Tutorials/whatiswot/12-Applications in Industrial and Agriculture Systems/README.md

```diff
@@ -0,0 +1,30 @@
+# Applications in Industrial and Agriculture Systems
+
+## Table of Contents
+
+- Brief Introduction
+- Industrial Systems
+- Agriculture Systems
+- Cross-domain Collaboration
+
+## Preliminary Script
+
+- Industrial applications of WoT involves more complex use cases than smart homes.
+
+- In industrial systems, devices from different manufacturers need to work together for a single use case. WoT supports this by accommodating various industry protocols, including legacy ones, ensuring seamless integration. The use of Thing Descriptions (TDs) allows devices to self-describe their capabilities, reducing the engineering effort needed to make them compatible.
+
+### Industrial systems
+
+- Industrial applications may resemble smart home applications but have broader requirements, such as advanced monitoring of connected manufacturing equipment as well as the manufactured products and prediction of possible machine failure.
+
+- Mention different protocols in factories and show TDs
+
+### Agriculture systems
+
+- Another domain-specific application is in agriculture systems.
+
+- With WoT, soil conditions can be monitored and optimal plans for watering and fertilizing can be created. Depending on the need, agricultural devices can be controlled remotely. Thus, the quality and quantity of the harvest can be increased.
+
+### Cross-domain Collaboration
+
+- In cross-domain collaborations, each system involves other systems in other domains. For example, Smart Factory with Smart City, Smart City with Smart Home. In these models, information can be exchanged directly between systems in a peer-to-peer manner or indirectly via collaboration platforms.
```

Comment:
In the end, we should summarize the benefits we mentioned above: domain agnostic, multi protocol, open standards, resource constrained devices or cloud.